src/recognizers/tap.js

```diff
@@ -21,38 +20,39 @@ import getDistance from '../inputjs/get-distance';
  * @constructor
  * @extends Recognizer
  */
-function TapRecognizer() {
-  Recognizer.apply(this, arguments);
-
-  // previous time and center,
-  // used for tap counting
-  this.pTime = false;
-  this.pCenter = false;
-
-  this._timer = null;
-  this._input = null;
-  this.count = 0;
-}
+export default class TapRecognizer extends Recognizer {
+  constructor() {
+    super(...arguments);
+    // previous time and center,
+    // used for tap counting
+    this.pTime = false;
+    this.pCenter = false;
+
+    this._timer = null;
+    this._input = null;
+    this.count = 0;
+  }
 
-inherit(TapRecognizer, Recognizer, {
   /**
    * @private
    * @namespace
    * @memberof PinchRecognizer
    */
-  defaults: {
-    event: 'tap',
-    pointers: 1,
-    taps: 1,
-    interval: 300, // max time between the multi-tap taps
-    time: 250, // max time of the pointer to be down (like finger on the screen)
-    threshold: 9, // a minimal movement is ok, but keep it low
-    posThreshold: 10 // a multi-tap can be a bit off the initial position
-  },
+  get defaults() {
```

Comment:
this should stay as a prop, with ES6 classes you can set props from within the constructor method.